tools/validate/validate.cpp

```diff
@@ -156,6 +156,7 @@ int main() {
         ".git"sv,
         ".vs"sv,
         ".vscode"sv,
+        "__pycache__"sv,
         "llvm-project"sv,
         "out"sv,
         "vcpkg"sv,

```

Comment:
Do you need to add `.pyc`, `.pyd`, and `.pyo` to `skipped_extensions`?

Reply:
I'm pretty sure those files can only exist under __pycache__ but I don't see why not.

Reply:
If they can only exist under that directory, then I think it would be better to leave validate.cpp mentioning only the directory, and instead simplify the .gitignore.